documentation/architecture/index.md

```diff
@@ -9,30 +9,103 @@ tags:
+
+### Networking
+
+#### Overview
+The networking is built on a principle of centralized on-premises and internet ingress/egress, while enforcing data plane isolation between workloads in different environments. Connectivity to on-premises environments, internet egress, shared resources and AWS APIs are mediated at a central point of ingress/egress via the use of a Transit Gateway. Consider the following overall network diagram:
+
+![Network Architecture](https://raw.githubusercontent.com/aws-samples/landing-zone-accelerator-on-aws-for-tse-se/35a3f853555eeb9231fc781cbeeef018b67dfb88/architecture-doc/images/network_architecture.drawio.png "Network Architecture")
+
+#### Perimeter
+The perimeter VPC hosts our perimeter security services. The Perimeter VPC is used to control the flow of traffic between AWS Accounts and external networks for IaaS workloads: both public and in some cases private.
+
+This VPC hosts AWS Network Firewall and/or 3rd party Next Generation Firewalls (NGFW) that provide perimeter security services including virus scanning / malware protection, Intrusion Protection services, TLS Inspection and Web Application Firewall protection.
+
+Note that this VPC is in its own isolated account, separate from Shared Network, in order to facilitate networking and security 'separation of duties'. Internal networking teams may administer the cloud networks in Shared Network without being granted permission to administer the security perimeter itself.
+
+#### Workload VPCs
+The workload VPCs are where line of business applications ultimately reside, segmented by environment (Dev, Test, Prod, etc).
+
+![Workload VPC](https://raw.githubusercontent.com/aws-samples/landing-zone-accelerator-on-aws-for-tse-se/35a3f853555eeb9231fc781cbeeef018b67dfb88/architecture-doc/images/workload.drawio.png "Workload VPC")
+
+Security groups are recommended as the primary data-plane isolation mechanism between applications that may coexist in the same VPC. It is anticipated that unrelated applications would coexist in their respective tiers without ever permitting east-west traffic flows.
+
+The following subnets will be defined:
+
+* **TGW subnet:** This subnet hosts the elastic-network interfaces for the TGW attachment. A /27 subnet is sufficient.
+* **Web subnet:** This subnet hosts front-end or otherwise 'client' facing infrastructure. A /20 or larger subnet is recommended to facilitate auto-scaling.
+* **App subnet:** This subnet hosts app-tier code (EC2, containers, etc). A /19 or larger subnet is recommended to facilitate auto-scaling.
+* **Data subnet:** This subnet hosts data-tier code (RDS instances, ElastiCache instances). A /21 or larger subnet is recommended.
+* **Mgmt subnet:** This subnet hosts bastion or other management instances. A /21 or larger subnet is recommended.
```

Comment:
```suggestion
* **Management subnet:** This subnet hosts bastion or other management instances. A /21 or larger subnet is recommended.
```

While accepted as a shorthand, I think for this kind of documentation it reads better as longhand?

Reply:
> While accepted as a shorthand, I think for this kind of documentation it reads better as longhand?

Agree, although if this is just to make the AWS documentation available on our pages, AWS refer to it as a Mgmt subnet in their architecture docs.

Reply:
I think it's also because that's the actual name of the subnet, I'll double check.

Reply:
Checked this and it creates a subnet per VPC but the name is Mgmt e.g. Dev-Mgmt-A, Central-Mgmt-A. @LiamMacP are you happy to resolve the conversation?

Reply:
Yep resolved 👍🏽